.github/workflows/test-integrations-data-processing.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.5","3.7","3.8","3.9","3.11","3.12"]
```

Comment:
Shouldn't we keep testing any 1.x changes on these Python versions to ensure we remain compatible?

Reply:
This whole step should actually not be there anymore at all -- it's testing `latest`. Looks like I missed some `latest` somewhere in tox.ini. Will recheck this.

But to answer more generally -- the goal is to get rid of all the `-latest` jobs and instead test additional pinned versions (stuff that was latest in the past) in the `-pinned` group.

Reply:
Yep, missed some stuff: https://github.com/getsentry/sentry-python/pull/3341/commits/1f1256c706f0066a8e3f88cad5d7af034665bc29 Fixed now.